app/assets/javascripts/application.js

```diff
@@ -20,4 +20,10 @@
 //= require nprogress
 //= require nprogress-turbolinks
 //= require main
-//= require jquery.sidr.min
\ No newline at end of file
+//= require jquery.sidr.min
+
+$(document).ajaxError(function (e, xhr, settings) {
```

Comment:
@rohitpaulk 
Not sure I got that correctly, but see here:
[.ajaxError()](https://api.jquery.com/ajaxError/)
> Note: This handler is not called for cross-domain script and cross-domain JSONP requests.